File: cookbooks/python/quickstart/01_hello_agent.py
"""
01 - Hello Agent
Sandboxed AI agent.
"""
import asyncio
from dotenv import load_dotenv
from evolve import Evolve

load_dotenv()

async def main():
    # Auto-resolves EVOLVE_API_KEY from environment
    agent = Evolve()

    await agent.run(
        prompt="""
            Research the latest developments in AI agents.
            Generate a brief report summarizing the top 3 findings.
        """
    )

    # Retrieve files from sandbox output/ folder
    output = await agent.get_output_files()
    print(list(output.files.keys()))

    await agent.kill()

if __name__ == "__main__":
    asyncio.run(main())

File: cookbooks/typescript/quickstart/01-hello-agent.ts
/**
 * 01 - Hello Agent
 * Sandboxed AI agent.
 */
import "dotenv/config";
import { Evolve } from "@evolvingmachines/sdk";

// Auto-resolves EVOLVE_API_KEY from environment
const agent = new Evolve();

await agent.run({
    prompt: `
        Research the latest developments in AI agents.
        Generate a brief report summarizing the top 3 findings.
    `,
});

// Retrieve report files from sandbox output/ folder
const output = await agent.getOutputFiles();
console.log(Object.keys(output.files));

await agent.kill();
